gnomad_qc/v3/sample_qc/sample_qc.py

```diff
@@ -530,11 +558,17 @@ def assign_pops(
         pop_pca_scores_ht,
         pc_cols=pop_pca_scores_ht.scores[:n_pcs],
         known_col="training_pop",
+        output_col=pop_field,
         min_prob=min_prob,
     )
 
+    # Set "<NA>" string to missing
+    pop_ht = pop_ht.annotate(
+        training_pop=hl.or_missing(pop_ht.training_pop != "<NA>", pop_ht.training_pop)
```

Comment:
I'm not following why this is needed. I would have assumed before being put into `assign_population_pcs` `training_pop` would either be missing or defined to a global or subpop. Is it missing before `assign_population_pcs` and somehow set to a string in `assign_population_pcs`?